pkg/proxyagent/agent/agent.go

```diff
@@ -231,7 +294,83 @@ func CustomSignerWithExpiry(customSignerName string, caKey, caData []byte, durat
 
 const (
 	ApiserverNetworkProxyLabelAddon = "open-cluster-management.io/addon"
-
-	AgentSecretName   = "cluster-proxy-open-cluster-management.io-proxy-agent-signer-client-cert"
-	AgentCASecretName = "cluster-proxy-ca"
+	AgentSecretName                 = "cluster-proxy-open-cluster-management.io-proxy-agent-signer-client-cert"
+	AgentCASecretName               = "cluster-proxy-ca"
 )
+
+func filterMCS(managedClusterSets []clusterv1beta1.ManagedClusterSet, clusterlabels map[string]string) ([]clusterv1beta1.ManagedClusterSet, error) {
```

Comment:
how about return the map[string]clusterv1beta1.ManagedClusterSet directly

Reply:
Updated, saved one turn of iter.